src/plopp/functions/slicer.py

```diff
@@ -68,6 +72,16 @@ def __init__(
                 f"were not found in the input's dimensions {ds.dims}."
             )
 
+        if autoscale == 'fixed':
+            if ('vmin' in kwargs) or ('vmax' in kwargs):
+                raise ValueError(
+                    'Slicer plot: when autoscale is set to "fixed", vmin and/or vmax '
+                    'cannot be specified.'
+                )
```

Comment:
Not sure this is necessary or convenient? Say I want to first see the full range, then manually set a maximum, or play with the settings. I think the `autoscale` could still apply to the other (vmin or vmax) that was not user-provided. If both are provided, maybe just warn, instead of raising?

Otherwise you could argue that also the `auto` or `grow` settings should raise if manual limits are given.